guides/common/modules/ref_json-response-metadata.adoc

```diff
@@ -0,0 +1,17 @@
+[id="json-response-metadata"]
+= JSON response metadata
+
+{Project} API responses contain the following metadata fields:
+
+* `total` {endash} The total number of objects without any search parameters.
```

Comment:
```suggestion
`total`:: The total number of objects without any search parameters.
```
Perhaps we should use a description list for these instead of an itemized list with en dashes.

Reply:
Implemented. Is that what you had in mind?

Reply:
Yes, the source looks good. Unfortunately, surge is timing out again. I'll try to check the build on Monday.